### PR TITLE
Stalagmite Changes

### PR DESCRIPTION
**Pull request by rosieposieeee**
## About the PR

image
304585648-b962ef07-58f5-4623-99d0-87cb975c7064
304586021-9f7bc676-9a4e-4df9-a26a-185beaf6ca49

Adds grey stalagmites.

Makes stalagmites significantly easier to break.

Adds a satisfying glass breaking sound to stalagmites.

## Why / Balance

These stalagmites are really pretty, and I'd like to use them in a map (see second & third screenshots) but a) there wasn't a matching set for the grey cave tile, and b) they're so absurdly strong, it takes like a hundred hits with a drill to break them. Hard to justify mapping with them when they are such an absurd obstacle. In real life, stalagmites are pretty fragile, and you have to be careful with them, so this is very silly.

They're very fun to smash through now.

## Media

- [X] I have added screenshots/videos to this PR showcasing its changes ingame, **or** this PR does not require an ingame showcase

**Changelog**
:cl:
- add: Added grey stalagmites and made all stalagmites weaker and more fun to break.

### CODE REVIEW

**Comment by TheShuEd** [comment]
maybe not glass sound? 

smth like this
https://freesound.org/people/SoundCollectah/sounds/109360/

File: Resources/Prototypes/Entities/Objects/Decoration/flora.yml [hunk 1]
```diff
@@ -217,6 +228,57 @@
     sprite: Objects/Decoration/Flora/flora_stalagmite.rsi
     state: stalagmite6
 
+- type: entity
+  name: stalagmite
```
Comment (TheShuEd) [inline Resources/Prototypes/Entities/Objects/Decoration/flora.yml:232]: useless, inherited from brown stalagmites

File: Resources/Prototypes/Entities/Objects/Decoration/flora.yml [hunk 2]
```diff
@@ -217,6 +228,57 @@
...
+  description: Natural stone spikes.
```
Comment (TheShuEd) [inline Resources/Prototypes/Entities/Objects/Decoration/flora.yml:233]: useless

**Review by TheShuEd — CHANGES_REQUESTED** [review]

**Comment by rosieposieeee** [comment]
> maybe not glass sound?
> 
> smth like this https://freesound.org/people/SoundCollectah/sounds/109360/

yeah, i'm sure there's something better... that sound is a little long though, so probably not that one. will get to the other stuff, holdovers i forgot to fix from when this was a downstream PR

**Comment by TheShuEd** [comment]
useless only name and description field
image

**Comment by rosieposieeee** [comment]
> useless only name and description field image

oops lol. sorry i'm juggling changes to like four prs right now haha i need to be more careful.

**Review by TheShuEd — APPROVED** [review]

**Pull request merged**